rackcity/views/it_instance_views.py

```diff
@@ -1,19 +1,48 @@
 # from rest_framework.parsers import JSONParser
 from django.http import HttpResponse, JsonResponse
 from rackcity.models import ITInstance
-from rackcity.api.serializers import ITInstanceSerializer
+from rackcity.api.serializers import (
+    ITInstanceSerializer,
+    RecursiveITInstanceSerializer,
+)
+from rest_framework.decorators import permission_classes, api_view
+from rest_framework.permissions import IsAuthenticated, IsAdminUser
+from rest_framework.parsers import JSONParser
+from rest_framework.pagination import PageNumberPagination
+from http import HTTPStatus
 
 
+@api_view(['GET'])
+@permission_classes([IsAuthenticated])
 def instance_list(request):
```

Comment:
This will be deprecated eventually right? Fine with it for now though

Reply:
Indeed. Keeping it so we don't break existing endpoints..........yet